### DISCUSSION TITLE
Preferred import style for python examples

### DISCUSSION THREAD

**Discussion by michael-c-brennan** [MeasureTransport/MParT#180] [Polls]
As mentioned in #145, how we import MParT in our python examples will most likely become the de facto standard for using the python bindings. How should we import MParT in the python examples?

**Comment by rubiop** [comment]
I like `mt` or `mpart` the most, I don't know if it is a regularly used library but I have been using the `multiprocessing` library and the import is usually `mp`. `from mpart import * ` can be good as the code will looks exactly like matlab but I am not sure how the general opinion is for this type of import.
